client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -286,7 +286,7 @@ class ThemeEnhancements extends Component {
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
 			<div>
-				<SettingsSectionHeader title={ translate( 'Theme Enhancements' ) } />
+				<SettingsSectionHeader title={ translate( 'Theme enhancements' ) } />
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.